src/event-actions/on-app-started.ts

```diff
@@ -33,6 +33,8 @@ export async function onAppStarted() {
 
   const initialUrl = await Linking.getInitialURL();
 
+  Alert.alert('initialUrl', initialUrl ?? '');
+
```

Comment:
I think it's better move it to a test screen

Reply:
nope =) I need test these as early as possible =) And in next build it will be removed